cli/README.md

```diff
@@ -1 +1,74 @@
 # Xata CLI
+
+## Installing
+
+The recommended way of installing the CLI is by installing it as an npm package globally:
+
+```bash
+npm install @xata.io/cli -g
+```
+
+Now you can just run `xata --help` to see all the available commands.
+
+It's perfectly fine to install it locally in your project if you wish:
+
+```bash
+npm install @xata.io/cli --save-dev
+```
+
+## Authentication, API keys and projects
+
+Almost all commands can work in two different modes: globally or scoped to a project. You can configure a project with `xata init`. This will create a `.xatarc` file (although `.xatarc.json` and `package.json` are also supported) and an `.env` file.
+
+- The project configuration file contains some preferences and information, including the `databaseURL`.
+- The `.env` file contains an API key and optionally a fallback branch name. More about this later.
```

Comment:
I'd personally remove the "more about this later".

Also maybe add here a mention that this file shouldn't be checked out into git. Maybe you added it after, doing a linear reading. In any case, I think it would be worth mentioning that now.

Question: We would probably want to git check out the fallback branch. Why didn't we add this to the project config?

Reply:
> We would probably want to git check out the fallback branch. Why didn't we add this to the project config?

Because the fallback branch can be different between people in the same project. That's even the ideal scenario, because you don't want to share the same dataset with another teammate. That could be a bit chaotic.